app/views/coronavirus_landing_page/components/shared/_country_section.html.erb

```diff
@@ -6,6 +6,11 @@
       link_to(link["label"],link["url"], class: "govuk-link")
     end
   %>
+  <% if guidance["intro"].present? %>
+    <p class="govuk-body">
+      <%= render_govspeak(guidance["intro"]) %>
+    <p>
+   <% end %>
```

Comment:
I think this should be moved outside of this `if` as if there are no links this won't show

Reply:
Right, so take this outside of this `<% if guidance.present? && guidance["links"].present? %>` check? So if the other links aren't present this paragraph will still render?

I'm wondering if the level of abstraction present in this partial could be reduced so there's editable content but it's more reflective of the rendered markup.